frontend/kesaseteli/employer/src/pages/thankyou.tsx

```diff
@@ -0,0 +1,58 @@
+import { IconPlus } from 'hds-react';
+import useApplicationApi from 'kesaseteli/employer/hooks/application/useApplicationApi';
+import { GetStaticProps, NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { useTranslation } from 'next-i18next';
+import Container from 'shared/components/container/Container';
+import withAuth from 'shared/components/hocs/withAuth';
+import Layout from 'shared/components/Layout';
+import getServerSideTranslations from 'shared/i18n/get-server-side-translations';
+
+import { $Notification, $PrimaryButton } from '../components/application/login.sc';
+
+const ThankYouPage: NextPage = () => {
+    const { t } = useTranslation();
+    const router = useRouter();
+    const { application } = useApplicationApi();
+
+    const createNewApplicationClick = (): void => {
+        void router.push("/");
+    };
+
+    if (!application || application.status === "draft") {
+        return null
```

Comment:
There we could redirect to application page if status is draft and if missing, then redirect to index page.